Tools/build/compute-changes.py

```diff
@@ -98,8 +103,14 @@ def process_changed_files(changed_files: Set[Path]) -> Outputs:
     run_tests = False
     run_ci_fuzz = False
     run_docs = False
+    run_windows = True
     run_windows_msi = False
 
+    # Check if changes are only to configuration/makefile files
+
+    if changed_files and all(file.name in LINUX_CONFIG_FILES for file in changed_files):
+        run_windows = False
```

Comment:
This double iteration is inefficient.

Reply:
Addressed in da444eb53178016eba8b7373b3ad3762318ef745